src/main/resources/templates/progress/stage.html

```diff
@@ -34,22 +34,22 @@ <h2>Týmy:</h2>
                     <td th:text="${team.numOfMembers}" align="center">8</td>
                     <td th:text="${team.pin}" align="center">9876</td>
                     <td align="center">
-                        <a href="#" class="btn btn-default btn-lg"
+                        <a href="#" class="btn btn-default btn-lg" th:onclick="return confirm('Opravdu změnit stav šifry?')"
```

Comment:
Tady bych byl rad, kdyby bylo i napsane pro jaky tym a na jaky stav. Napriklad:
`th:onclick="return confirm('Opravdu chcete PRESKOCIT sifru pro tym ${team.getName()}?')`. Jde mi o to, ze kdyz se nekdo preklikne a zmackne to tlacitko pro jiny tym, tak aby v tom dialogu mel napsane, co vlastne dela.